src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,39 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+class Block{
+	public static int blockNumber = 123;
+	public Block(int blockNumber){
+		Block.blockNumber = 123;
```

Comment:
The expected behaviour of the constructor is assigning the blockNumber in the argument to the blockNumber in the class. You have simply assigned 123 as the blockNumber to every instance of Block. Which is incorrect. 

Consider an implementation such as https://github.com/nus-cs2103-AY1617S2/addressbook-level2/pull/318/files